python/ray/autoscaler/vsphere/defaults.yaml

```diff
@@ -52,8 +52,9 @@ auth:
 # The node config specifies the launch config and physical instance type.
 available_node_types:
     ray.head.default:
-        # For example, {"CPU": 4, "Memory": 8192}
-        resources: {}
+        # The node type's CPU and GPU resources are by default the same as the frozen VM.
+        # You can override the resources here.
+        resources: {"CPU": 2}
```

Comment:
I think when the user uses a cluster launcher `YAML` file and omits a field, Ray will automatically fill in the value from this `defaults.yaml`.  Do we really want to automatically fill in `"CPU"": 2` if the user doesn't specify `resources`?

Reply:
Yes. To keep the frozen VM's footprint small, the frozen VM is configured with only 1 CPU. So if not specified, the created head and worker node VMs will inherit this setting to be 1 CPU for each.

I checked the defaults.yaml for other providers. Looks like aws uses m5.large which is 2 vCPU. And azure and gcp both explicitly specify 2 CPUs. So we also put `"CPU"": 2` for ours.